include/boost/gil/image_processing/threshold.hpp

```diff
@@ -45,7 +45,7 @@ void threshold_impl(SrcView const& src_view, DstView const& dst_view, Operator c
         typename color_space_type<DstView>::type
     >::value, "Source and destination views must have pixels with the same color space");
 
-    //iterate over the image chaecking each pixel value for the threshold
+    //iterate over the image checking each pixel value for the threshold
```

Comment:
Good catch! Could you submit this as a separate bugfix PR?

Reply:
Yeah sure, will do